packages/web/pages/index.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from "react";
+import Header from "../components/Header";
 import { Wrapper } from "../components/Wrapper";
 
 const Index = () => {
   return (
     <Wrapper>
-      <h1>Welcome to Next.js!</h1>
+      <Header height={100} />
```

Comment:
Wiem, że to akurat nie Ty dodałeś, ale komponent `Wrapper` powinien być wewnątrz `Header`. Wtedy po prostu używamy komponentu header i nie martwimy się o zbędny szum wokół.